scripts/install-service.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -e 
+
+# Change to script directory for predictable behaviour
+cd "$(dirname "$0")"
+
+# Check whether virtual environment is active
+if [ -z "${VIRTUAL_ENV}" ] && [ "${FORCE}" != "true"  ]; then
+    echo "Warning: Virtual environment not detected! It is highly recommended to install this program to run under a virtual environment. Please activate your virtual environment and run this script again."
+    echo "If you're certain you'd like to continue without a virtual environment, you can run again with 'FORCE=true'." 
+    exit 1
+fi
+
+# Install poglink python package
+pip install poglink
+
+# Obtain path to python executable
+if [ -z "${VIRTUAL_ENV}" ]; then
+    PYTHON_EXECUTABLE="$(which python3)"
+else 
+    PYTHON_EXECUTABLE="${VIRTUAL_ENV}/bin/python3"
+fi 
```

Comment:
This is what I came up with to help encourage usage within a virtual environment, while not explicitly preventing someone from ignoring that advice. 

I use `pyenv` and it should work equally well with `venv`. Please do test if you'd like though.